src/main/java/edu/ksu/canvas/attendance/submitter/AssignmentSubmitter.java

```diff
@@ -174,13 +174,13 @@ private double calculateStudentGrade(AttendanceAssignment attendanceAssignment,
         if (isSimpleAttendance) {
             int totalClasses = getTotalSimpleClasses(entry);
             totalClasses = totalClasses == 0? 1 : totalClasses;
-            return ((entry.getTotalClassesPresent() * (attendanceAssignment.getPresentPoints() / 100) +
-                    entry.getTotalClassesTardy() * (attendanceAssignment.getTardyPoints() / 100) +
-                    entry.getTotalClassesExcused() * (attendanceAssignment.getExcusedPoints() / 100) +
-                    entry.getTotalClassesMissed() * (attendanceAssignment.getAbsentPoints() / 100)) / totalClasses) * attendanceAssignment.getAssignmentPoints();
+            return ((entry.getTotalClassesPresent() * (Double.parseDouble(attendanceAssignment.getPresentPoints()) / 100) +
+                    entry.getTotalClassesTardy() * (Double.parseDouble(attendanceAssignment.getTardyPoints()) / 100) +
+                    entry.getTotalClassesExcused() * (Double.parseDouble(attendanceAssignment.getExcusedPoints()) / 100) +
+                    entry.getTotalClassesMissed() * (Double.parseDouble(attendanceAssignment.getAbsentPoints()) / 100)) / totalClasses) * Double.parseDouble(attendanceAssignment.getAssignmentPoints());
         } else {
 
-            return ((100 - entry.getPercentCourseMissed()) / 100) * attendanceAssignment.getAssignmentPoints();
+            return ((100 - entry.getPercentCourseMissed()) / 100) * Double.parseDouble(attendanceAssignment.getAssignmentPoints());
         }
```

Comment:
maybe another helper method would help reduce the repeated logic for calculating the point percentage that returns (total * (points/100))...